drivers/can/Kconfig.sam

```diff
@@ -0,0 +1,22 @@
+# SAM CAN configuration options
+# Copyright (c) 2021 Alexander Wachter
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ATMEL_SAM_CAN := atmel,sam-can
+
+config CAN_SAM
+	bool "Atmel SAM CAN driver"
+	default $(dt_compat_enabled,$(DT_COMPAT_ATMEL_SAM_CAN))
+	select CAN_MCAN
+
+if CAN_SAM
+
+config CAN_SAM_CKDIV
+	int "Clock divider"
+	range 0 255
+	default 0
+	depends on CAN_SAM
+	help
+	  Clock divider for the MCAN core clock.
```

Comment:
Shouldn't this be configured via DTS instead?

Reply:
Not sure. This config applies to each controller.
For stm32 it is also a Kconfig